HW456/HW56/Accounts/IncreaseAccount.cs

```diff
@@ -0,0 +1,79 @@
+namespace HW456.HW5.Accounts;
+
+public class IncreaseAccount
+{
+    public string AccountNumber { get; set; }
+    public string OwnerName { get; set; }
+    public virtual decimal Balance { get; set; }
+    public bool IsActive { get; set; }
+
+    public virtual bool IncreaseBalance(decimal sum)
+    {
+        if (IsActive)
+        {
+            Balance += sum;
+            return true;
+        }
+        else
+        {
+            Console.WriteLine("Operation is not available");
+            return false;
+        }

```

Comment:
Вообще тут else лишний, т.к. первое условие прерывает дальнейшее выполнение.

И тут стоит изменить немного код: обычно, если есть какой то негативный сценарий, который прерывает дальнейшее выполнение кода, то его переносят в самое начало метода, так вложенность последующего сценария понижается.
Имею ввиду сделать так:

```suggestion
        if (!IsActive)
        {
            Console.WriteLine("Operation is not available");
            return false;
        }

        Balance += sum;
        return true;
```

Reply:
поправила